<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fourn</class>
 <widget class="QDialog" name="fourn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1448</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>2061</width>
     <height>611</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>140</y>
       <width>551</width>
       <height>291</height>
      </rect>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>80</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>80</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_username">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>60</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit__password">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>130</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_login">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>210</y>
        <width>160</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>20</y>
       <width>1181</width>
       <height>561</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTabWidget::pane {
  border: 0px solid #2CBB63; /* Green contour */
  border-radius: 8px;
  background-color: white;
}

QTabWidget::tab-bar {
  alignment: center;
}

QTabBar::tab {
  background-color: white;
  color: black; /* Black text color */
  border: none; /* Remove the border */
  padding: 14px 25px;
  font-size: 12px;
  font-weight: bold;
  min-width: 120px;
  qproperty-drawBase: 0;
}

QTabBar::tab:selected {
  background-color: white; /* Green color */
  color: black; /* Black text color */
  border-bottom: 2px solid #2CBB63; /* Green underline */
}

QTabBar::tab:hover {
  background-color: white; /* Transparent background */
  color: black; /* Black text color */
  border-bottom: 4px solid #2CBB63; /* Green underline */
}

QTabBar::tab:selected:hover {
  background-color: white; /* Green color */
  color: black; /* Black text color */
}</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Ajouter Fornisseur</string>
      </attribute>
      <widget class="QPushButton" name="modifier">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>400</y>
         <width>160</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
 	border-radius:15px;
	color: white;
	background-color: rgb(0, 0, 255);
	}</string>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>140</y>
         <width>80</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
  padding: 5px;
  border-bottom: 2px solid #2CBB63; /* Grey underline */
  background-color: white; /* White background */
  color: black; /* Black text color */
  font-weight: bold; /* Bolder text */
  font-size:14px;
}

QLabel:hover {
  border-bottom-color: #2CBB63; /* Green underline color on hover */
  color: black; /* Dark green text color on hover */
}
</string>
       </property>
       <property name="text">
        <string>Old CIN:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>190</y>
         <width>160</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
  padding: 5px;
  border-bottom: 2px solid #2CBB63; /* Grey underline */
  background-color: white; /* White background */
  color: black; /* Black text color */
  font-weight: bold; /* Bolder text */
  font-size:14px;
}

QLabel:hover {
  border-bottom-color: #2CBB63; /* Green underline color on hover */
  color: black; /* Dark green text color on hover */
}
</string>
       </property>
       <property name="text">
        <string>EMAIL</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouter">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>400</y>
         <width>160</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: #2CBB63; /* Green color */
 	border-radius:15px;
	
	}
</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="l_CIN_3">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>140</y>
         <width>170</width>
         <height>25</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>113</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(105,118,132,255);

padding-bottom:7px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="l_email">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>190</y>
         <width>160</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(105,118,132,255);

padding-bottom:7px;
}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="l_CIN_2">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>40</y>
         <width>170</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(105,118,132,255);

padding-bottom:7px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>240</y>
         <width>160</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
  padding: 5px;
  border-bottom: 2px solid #2CBB63; /* Grey underline */
  background-color: white; /* White background */
  color: black; /* Black text color */
  font-weight: bold; /* Bolder text */
  font-size:14px;
}

QLabel:hover {
  border-bottom-color: #2CBB63; /* Green underline color on hover */
  color: black; /* Dark green text color on hover */
}
</string>
       </property>
       <property name="text">
        <string>Num-tel:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>40</y>
         <width>80</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
  padding: 5px;
  border-bottom: 2px solid #2CBB63; /* Grey underline */
  background-color: white; /* White background */
  color: black; /* Black text color */
  font-weight: bold; /* Bolder text */
  font-size:14px;
}

QLabel:hover {
  border-bottom-color: #2CBB63; /* Green underline color on hover */
  color: black; /* Dark green text color on hover */
}
</string>
       </property>
       <property name="text">
        <string>CIN:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>340</y>
         <width>160</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
  padding: 5px;
  border-bottom: 2px solid #2CBB63; /* Grey underline */
  background-color: white; /* White background */
  color: black; /* Black text color */
  font-weight: bold; /* Bolder text */
  font-size:14px;
}

QLabel:hover {
  border-bottom-color: #2CBB63; /* Green underline color on hover */
  color: black; /* Dark green text color on hover */
}
</string>
       </property>
       <property name="text">
        <string>Sexe:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>40</y>
         <width>160</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
  padding: 5px;
  border-bottom: 2px solid #2CBB63; /* Grey underline */
  background-color: white; /* White background */
  color: black; /* Black text color */
  font-weight: bold; /* Bolder text */
  font-size:14px;
}

QLabel:hover {
  border-bottom-color: #2CBB63; /* Green underline color on hover */
  color: black; /* Dark green text color on hover */
}
</string>
       </property>
       <property name="text">
        <string>Prenom:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="l_nomentreprise">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>290</y>
         <width>160</width>
         <height>26</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {
    border-radius: 9px;
    border: 2px solid #000000;
   background-color: #2CBB63; /* Green color */
    color: #000000; /* Couleur du texte */
}
</string>
       </property>
       <item>
        <property name="text">
         <string>Choisi</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Kalkhoff</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tesla</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hyundai</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mazda</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Beaufort</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Trek</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>140</y>
         <width>160</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
  padding: 5px;
  border-bottom: 2px solid #2CBB63; /* Grey underline */
  background-color: white; /* White background */
  color: black; /* Black text color */
  font-weight: bold; /* Bolder text */
  font-size:14px;
}

QLabel:hover {
  border-bottom-color: #2CBB63; /* Green underline color on hover */
  color: black; /* Dark green text color on hover */
}
</string>
       </property>
       <property name="text">
        <string>CIN:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>290</y>
         <width>160</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
  padding: 5px;
  border-bottom: 2px solid #2CBB63; /* Grey underline */
  background-color: white; /* White background */
  color: black; /* Black text color */
  font-weight: bold; /* Bolder text */
  font-size:14px;
}

QLabel:hover {
  border-bottom-color: #2CBB63; /* Green underline color on hover */
  color: black; /* Dark green text color on hover */
}
</string>
       </property>
       <property name="text">
        <string>Nom de lentreprise : </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="l_prenom">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>40</y>
         <width>160</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
  padding: 0;
  border: none; /* No border */
  background-color: transparent; /* Transparent background */
  color: black; /* Black text color */
  border-bottom: 2px solid #808080; /* Grey underline */
}

QLineEdit:hover {
  border-bottom: 3px solid #2CBB63; /* Green underline on hover */
  font-weight: bold; /* Bolder text */
  padding-bottom: 2px; /* Increase bottom padding */
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="l_CIN">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>140</y>
         <width>160</width>
         <height>25</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>113</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(105,118,132,255);

padding-bottom:7px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>90</y>
         <width>160</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
  padding: 5px;
  border-bottom: 2px solid #2CBB63; /* Grey underline */
  background-color: white; /* White background */
  color: black; /* Black text color */
  font-weight: bold; /* Bolder text */
  font-size:14px;
}

QLabel:hover {
  border-bottom-color: #2CBB63; /* Green underline color on hover */
  color: black; /* Dark green text color on hover */
}
</string>
       </property>
       <property name="text">
        <string>Nom:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier_2">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>130</y>
         <width>190</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(0, 0, 255);
 	border-radius:15px;
    color: white;
	}</string>
       </property>
       <property name="text">
        <string>Afficher avant mofication</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="l_tel">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>240</y>
         <width>160</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(105,118,132,255);

padding-bottom:7px;
}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="l_nom">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>90</y>
         <width>160</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(105,118,132,255);

padding-bottom:7px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="supprimer">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>30</y>
         <width>190</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: rgb(255, 0, 0);
 	border-radius:15px;
	
	}</string>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
       <property name="icon">
        <iconset theme="poubelle">
         <normaloff>../Projetcpp</normaloff>../Projetcpp</iconset>
       </property>
      </widget>
      <widget class="QRadioButton" name="Homme">
       <property name="geometry">
        <rect>
         <x>246</x>
         <y>347</y>
         <width>97</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Homme</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="Femme">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>347</y>
         <width>97</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Femme</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>230</y>
         <width>241</width>
         <height>191</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="image.qrc">:/img/alleco.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Afficher Fournisseur</string>
      </attribute>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>110</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Recherche :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="l_entreprise">
       <property name="geometry">
        <rect>
         <x>970</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(105,118,132,255);

padding-bottom:7px;
}</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButto_stat">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>120</y>
         <width>200</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: #2CBB63; /* Green color */
 	border-radius:15px;
	
	}
</string>
       </property>
       <property name="text">
        <string>Statistique</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>180</width>
         <height>40</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="PDF">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>70</y>
         <width>200</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: #2CBB63; /* Green color */
 	border-radius:15px;
	
	}
</string>
       </property>
       <property name="text">
        <string>PDF</string>
       </property>
      </widget>
      <widget class="QPushButton" name="qr_code">
       <property name="geometry">
        <rect>
         <x>860</x>
         <y>120</y>
         <width>190</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: #2CBB63; /* Green color */
 	border-radius:15px;
	
	}
</string>
       </property>
       <property name="text">
        <string>QR CODE</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>180</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #2CBB63;
</string>
       </property>
       <item>
        <property name="text">
         <string>Trier les fournisseurs</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tri par Id</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tri par Nom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tri par NomEntreprise</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="codeQr">
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>170</y>
         <width>291</width>
         <height>271</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButto_afficher">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>120</y>
         <width>190</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: #2CBB63; /* Green color */
 	border-radius:15px;
	
	}
</string>
       </property>
       <property name="text">
        <string>Afficher</string>
       </property>
      </widget>
      <widget class="QPushButton" name="camera">
       <property name="geometry">
        <rect>
         <x>860</x>
         <y>70</y>
         <width>190</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: #2CBB63; /* Green color */
 	border-radius:15px;
	
	}
</string>
       </property>
       <property name="text">
        <string>Camera</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>70</y>
         <width>190</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  background-color: rgb(255, 0, 0);
 	border-radius:15px;
	
	}</string>
       </property>
       <property name="text">
        <string>Afficher Sur LCD</string>
       </property>
      </widget>
      <widget class="QTableView" name="tab_etud">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>741</width>
         <height>261</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableView {
  background-color: white; /* White background */
  color: black; /* Black text color */
  gridline-color: #2CBB63; /* Grid line color */
}

QTableView::item {
  padding: 5px; /* Add padding to table cells */
}

QTableView::item:selected {
  background-color: #2CBB63; /* Set the background color of selected cells */
  color: white; /* Set the text color of selected cells */
}

QHeaderView::section {
  background-color: #2CBB63; /* Set the background color of header sections */
  color: white; /* Set the text color of header sections */
  font-weight: bold; /* Bolder text */
}</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="image.qrc">:/img/alleco.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
